### ISSUE TITLE
Design Issue - Setting - write page

### ISSUE THREAD

**Issue by surapuramakhil** [apache/answer#816]
## Describe the bug

relevant discussion https://github.com/apache/incubator-answer/issues/815
https://github.com/apache/incubator-answer/issues/815#issuecomment-1974228257

Based on UI it has two sections that's what UI layout says to the users. something like this
![image](https://github.com/apache/incubator-answer/assets/9161543/92121b9d-466a-47ec-afd7-6eef1457207c)

**But this is what it is**
![image](https://github.com/apache/incubator-answer/assets/9161543/56a78614-630b-4a18-81ed-9713b3447b4e)

### Screenshots

If applicable, add screenshots or video to help explain your problem.

### Platform

- Device: [e.g. Desktop, Mobile]
- OS: [e.g. macOS]
- Browser and version: [e.g. Chrome, Safari]
- Version: [e.g. v1.2.0]

**Comment by zahash** [comment]
@surapuramakhil 

i don't think this can be changed. because every single admin page uses the same layout. 
not sure if there is no way to customize just one of them.

**Comment by surapuramakhil** [comment]
Ah, there were discussions on how to address this in https://github.com/apache/incubator-answer/issues/815
